ibm/mas_devops/roles/sls_cfg/README.md

```diff
@@ -0,0 +1,3 @@
+sls_cfg
```

Comment:
What is this new role doing?  What are the reasons for breaking this out into it's own role, please provide information in the PR / in this thread.

Reply:
This role is used to generate a configuration file incase of an existing sls.

Reply:
DIscussed in Slack, this is a different usecase, we will move all slscfg mgmt into a seperate role so that it is no longer duplicated in the sls_install role